code-challenges.test.js

```diff
@@ -13,10 +13,41 @@
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided.
+
+describe('stFunct1',  () => {
+  it('takes in an array and removes the first item shuffling the remaining content', () => {
+    expect(stFunct1(colors1)).toEqual(expect.arrayContaining(['blue', 'green', 'yellow', 'pink']))
+  })
+})
+
+describe('stFunct2', () => {
+  it('takes in an array and removes the first item shuffling the remaining content', () => {
+    expect(stFunct2(colors2)).toEqual(expect.arrayContaining(['saffron', 'aquamarine', 'periwinkle', 'indigo', 'ochre']))
+  })
+})
+// GOOD Failure
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   ReferenceError: stFunct1 is not defined
+// FAIL  ./code-challenges.test.js
+// ● Test suite failed to run
+
+//   ReferenceError: stFunct2 is not defined
+
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
+//Psudeocode: The function takes an array as input, then it creates a new array called newArray by extracting elements from colors1 starting from index 1 (excluding the first element). Then, it shuffles the elements in newArray using the .sort() method with a random comparison function (() => 0.5 - Math.random()). The stFunct function is called with colors1 as an argument, and the resulting shuffled array is stored in the finalArray variable. The finalArray is then displayed or printed to the console. The process is the same exact way for both array of colors (colors1, colors2), all that was changed was naming conventions. 
```

Comment:
As you said in the pseudo code, the process is the same for each variable, as it should be. This means there's no need to declare 2 separate functions for the 2 variables. Also, no need to store your function call in a variable like you did on line 47.